azerothcore/azerothcore-wotlk
# fix(Core/Spells): Everlasting Affliction should be considered as shad…

…ow spell.

Fixes #11624



## Issues Addressed:

- Closes #11624

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

1.Spec into [Everlasting Affliction](https://db.rising-gods.de/?search=everlasting+affliction)
2, Engage Nefarian
3, Wait for Paladin class call
4, Cast [Corruption](https://db.rising-gods.de/?spells=7.9.355&filter=na=Corruption)
5, Cast [Haunt](https://db.rising-gods.de/?spells=7.9.355&filter=na=Haunt) or [Shadow bolt](https://db.rising-gods.de/?spells=7.9.593&filter=na=Shadow%20bolt)
6. Notice that corruption is getting refreshed


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
